_docs-sources/guides/stay-up-to-date/cis/cis-1.5.0/deployment-walkthrough/step-3-update-the-account-baseline-modules.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_label: Update the Account Baseline modules
+---
+
+# Step 3: Update the Account Baseline modules
+
+Next, you will need to update the account baseline (landing zone) modules to the version compatible
+with v1.5.0 of the CIS AWS Foundations Benchmark. 
+
+We strongly recommend upgrading straight to at least [v0.42.9](https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/releases/tag/v0.42.9), instead of each minor version incrementally.
+
+If you are using Patcher, then it will upgrade to the latest available version.
+
+This guide assumes you are using at least [v0.40.0](https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/releases/tag/v0.40.0) of the CIS Service Catalog repo, which was released in August 2022.
+
+The account baseline modules had one breaking change between versions v0.40.0 and v0.42.9. We must manually run
+the migration steps before updating the module versions.
+
+- [v0.42.0](https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/releases/tag/v0.42.0): Added support for new AWS region (me-central-1 UAE) to multiregion modules.
+
+## 3.1 Add `me-central-1` to your `multi_region_common.hcl`
```

Comment:
Does Patcher do this automatically?

Reply:
@brikis98 not at present. We don't have a dedicated patch for this.

Reply:
Do we need to add that?

Reply:
(Side note: not a blocker for merging this PR, but something that can be done as a follow-up if we think it's necessary)

Reply:
@brikis98 I think we probably want this in the long-term, but I don't want to block launch. `terrapatch` currently doesn't have a dedicated command for this so we'd need to build one and its likely non-trivial. I think its worth adding an issue or something to the technical roadmap.